src/components/InfiniteList.js

```diff
@@ -59,7 +59,7 @@ export default class InfiniteList extends React.Component {
             {({ width }) => (
               <List
                 ref={this.hookRef(registerChild)}
-                className={styles.List}
+                className={`${styles.List && styles.List} ${customClassName}`}
```

Comment:
What do you mean here? `` `${styles.List && styles.List}` `` === `` `${styles.List}` `` I believe

Reply:
It should render the `styles.List `value only if exists and it's not undefined.
When using the `` notation, any undefined value would be printed anyway.